firefox-ios/Client/Frontend/Home/PocketFooterView.swift

```diff
@@ -29,12 +30,14 @@ class PocketFooterView: UICollectionReusableView, ReusableCell, ThemeApplicable
         label.font = FXFontStyles.Regular.caption1.scaledFont()
     }
 
-    private let learnMoreLabel: UILabel = .build { label in
-        label.text = .FirefoxHomepage.Pocket.Footer.LearnMore
-        label.isUserInteractionEnabled = true
-        label.adjustsFontForContentSizeCategory = true
-        label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.Pocket.footerLearnMoreLabel
-        label.font = FXFontStyles.Regular.caption1.scaledFont()
+    private let learnMoreButton: LinkButton = .build { button in
+        let buttonViewModel = LinkButtonViewModel(
+            title: .FirefoxHomepage.Pocket.Footer.LearnMore,
+            a11yIdentifier: AccessibilityIdentifiers.FirefoxHomepage.Pocket.footerLearnMoreLabel,
+            font: FXFontStyles.Regular.caption1.scaledFont()
+        )
+        button.configure(viewModel: buttonViewModel)
+        button.configuration?.contentInsets = NSDirectionalEdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0)
```

Comment:
thanks @DanielDervishi, can we set the contentInsets via the view model's initializer as well?